calico-cloud/networking/egress/egress-gateway-azure.mdx

```diff
@@ -0,0 +1,678 @@
+apiVersion: projectcalico.org/v3
+metadata:
+  name: peer-with-route-reflectors
+spec:
+  nodeSelector: all()
+  peerSelector: route-reflector == 'true'
+```
+
+Finally, Add the IP addresses of Azure Route Servers, usually there are two, as peers to route reflectors. For example, if the {{prodname}}
+cluster is in a subnet with 10.224.0.0 as network address, the 10.224.0.1 is, [by definition](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-networks-faq#are-there-any-restrictions-on-using-ip-addresses-within-these-subnets), the gateway and Azure Route Servers are deployed with
+ASN 65515 and IP addresses of 10.225.0.4 and 10.225.0.5, create two **BGPPeer** resources:
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: BGPPeer
+metadata:
+  name: azure-route-server-a
+spec:
+  peerIP: 10.225.0.4
+  reachableBy: 10.224.0.1
+  asNumber: 65515
+  keepOriginalNextHop: true
+  nodeSelector: route-reflector == 'true'
+  filters:
+    - export-egress-ips
+---
+apiVersion: projectcalico.org/v3
+kind: BGPPeer
+metadata:
+  name: azure-route-server-b
+spec:
+  peerIP: 10.225.0.5
+  reachableBy: 10.224.0.1
+  asNumber: 65515
+  keepOriginalNextHop: true
+  nodeSelector: route-reflector == 'true'
+  filters:
+    - export-egress-ips
+```
+
+:::note
+
+- Adding `routeReflectorClusterID` to a node spec will remove it from the node-to-node mesh immediately, tearing down the
+existing BGP sessions. Adding the BGP peering will bring up new BGP sessions. This will cause a short (about 2 seconds)
+disruption to dataplane traffic of workloads running in the nodes where this happens. To avoid this, make sure no
+workloads are running on the nodes, by provisioning new nodes or by running `kubectl drain` on the node (which may
+itself cause a disruption as workloads are drained).
+- It is important to set `keepOriginalNextHop: true` since route reflectors advertise routes on behalf of other nodes.
+Advertised routes to Azure Route Servers should have the original next hop otherwise the return packets will be sent to
+route reflectors, and get dropped.
+- It is mandatory to set `rechableBy` field set to the gateway of the subnet {{prodname}} cluster is running in for
+peering with Azure Route Servers to prevent BGP connection flapping. See [BGP peer](../../reference/resources/bgppeer.mdx) for more information.
+- Including `filters` to apply the BGP filter configured in the previous section, is optional.
+:::
+
+### Enable egress gateway support
+
+In the default **FelixConfiguration**, set the `egressIPSupport` field to `EnabledPerNamespace` or
+`EnabledPerNamespaceOrPerPod`, according to the level of support that you need in your cluster. For
+support on a per-namespace basis only:
+
+```batch
+kubectl patch felixconfiguration.p default --type='merge' -p \
+    '{"spec":{"egressIPSupport":"EnabledPerNamespace"}}'
+```
+
+Or for support both per-namespace and per-pod:
+
+```batch
+kubectl patch felixconfiguration default --type='merge' -p \
```

Comment:
We should be consistent with how we specify resources. Here we have `felixconfiguration`, a few lines above you have `felixconfiguration.p`.